app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```

Comment:
- [ ] As per this [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=You%20should%20follow%20these%20design%20guidelines%2C%20including%3A), Please follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:

- Colors.
- Typography: font face, size and weight.
- Layout: composition and space between elements.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/208915692-ea6c0c05-0d4d-4df2-bdad-45483bd7689f.png)  | ![image](https://user-images.githubusercontent.com/68796232/208915593-1df9f2f6-13d9-4fbe-9e35-880064eede9c.png)  |